aggregations-and-analytics/12_lag/12_lag.md

```diff
@@ -0,0 +1,43 @@
+# 12 Lag
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.13%2B-lightgrey)
```

Comment:
I've been checking the documentation but it looks like LAG was supported since Flink 1.9 (its listed at https://nightlies.apache.org/flink/flink-docs-release-1.9/dev/table/functions.html#aggregate-functions but not for the 1.8 documentation)

Reply:
Yep, that's now fixed